<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-QLGVSIUB" failures="2" tests="2"
	name="Default test(failed)" time="24.433" errors="0"
	timestamp="2024-11-12T21:27:39 IST">
	<testcase classname="webtest.DataDrivenTest" name="testdatadriven"
		time="12.476">
		<failure type="java.lang.AssertionError"
			message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at webtest.DataDrivenTest.testdatadriven(DataDrivenTest.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
	</testcase> <!--
	testdatadriven -->
	<testcase classname="webtest.DataDrivenTest" name="testdatadriven"
		time="10.916">
		<failure type="java.lang.AssertionError"
			message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at webtest.DataDrivenTest.testdatadriven(DataDrivenTest.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
	</testcase> <!--
	testdatadriven -->
</testsuite> <!--
Default test(failed) -->
